client/src/components/ReviewFilters/LanguageTypeFilter.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { LanguageTypeContainer, LanguageTypeTitle, LanguageTypeFlyoutMenu, DownArrow, FlyoutMenuText, FlyoutMenuStat, QuestionMarkImage, CustomizeButton } from '../../styled';
+
+const LanguageTypeFilter = ({ reviewStats, updateReviewFilters, questionMarkImage }) => (
+  <LanguageTypeContainer>
+    <LanguageTypeTitle>
+      Language
+      <DownArrow className="language_down_arrow">&#9662;</DownArrow>
+      <LanguageTypeFlyoutMenu className="language_type_flyout">
+        <form>
+          <input type="radio" className="all_languages_filter" value="all" name="languageType" onChange={(e) => { updateReviewFilters(e.target.value, 'languageType'); }} />
+          <FlyoutMenuText>All Languages</FlyoutMenuText>
+          <FlyoutMenuStat className="all_languages_count_stat">{`(${reviewStats.totalReviewCount})`}</FlyoutMenuStat>
```

Comment:
I noticed how you use a lot of classes in conjunction with styled-components. I'm interested to know why that is?

Reply:
I use classes for testing and for targeting elements that need to be displayed on hover!